metagraph/src/graph/representation/canonical_dbg.hpp

```diff
@@ -172,17 +156,19 @@ class CanonicalDBG : public DeBruijnGraph {
     mutable caches::fixed_sized_cache<node_index, std::vector<node_index>,
                                       caches::LRUCachePolicy<node_index>> parent_node_cache_;
 
-    // caches for the results of reverse_complement.
-
-    // cache the index of the reverse complement of a node
-    mutable caches::fixed_sized_cache<node_index, node_index,
-                                      caches::LRUCachePolicy<node_index>> rev_comp_cache_;
-
     // cache whether a given node is a palindrome (it's equal to its reverse complement)
     mutable caches::fixed_sized_cache<node_index, bool,
                                       caches::LRUCachePolicy<node_index>> is_palindrome_cache_;
 
     node_index reverse_complement(node_index node) const;
+
+    // find all parent nodes of node which are only represented in their reverse
+    // complement orientation in the underlying graph
+    void append_parent_nodes_using_node_rev_comp(node_index node, std::vector<node_index> &parents) const;
+
+    // find all child nodes of node which are only represented in their reverse
+    // complement orientation in the underlying graph
+    void append_child_nodes_using_node_rev_comp(node_index node, std::vector<node_index> &children) const;
```

Comment:
`which are only represented ...` is redundant. THe graph is primary, hence they can't be represented in both orientations in the underlying graph.

Maybe better renmae these methods to `append_next_rc_nodes`, and `append_prev_rc_nodes`?

Reply:
How about `find all parent nodes of node in the CanonicalDBG which are represented in the reverse complement orientation in the underlying primary graph`?